### PR TITLE
fix: make sure that the `__import_unsupported` patch is applied in Cloudflare as well

### PR DESCRIPTION
**Pull request by dario-piotrowicz**
this is needed to resolve https://github.com/opennextjs/opennextjs-cloudflare/issues/177

### CODE REVIEW

**Review by vicb — APPROVED** [review]
LGTM!

nit: maybe add a comment

**Comment by vicb** [comment]
@dario-piotrowicz would be nice to link to the issue on the cloudflare adapter in the PR description.
Thanks!

File: packages/open-next/src/build/edge/createEdgeBundle.ts [hunk 1]
```diff
@@ -130,21 +130,22 @@ globalThis.Buffer = Buffer;
 
 import {AsyncLocalStorage} from "node:async_hooks";
 globalThis.AsyncLocalStorage = AsyncLocalStorage;
+
+const defaultDefineProperty = Object.defineProperty;
```
[the remark this answers is not in the document]
Comment (dario-piotrowicz) [reply packages/open-next/src/build/edge/createEdgeBundle.ts:146]: comment added, please have a look

**Review by conico974 — APPROVED** [review]
LGTM Thanks.
Just resolve this https://github.com/opennextjs/opennextjs-aws/actions/runs/12351796922/job/34467482276?pr=678

File: packages/open-next/src/build/edge/createEdgeBundle.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (vicb) [reply packages/open-next/src/build/edge/createEdgeBundle.ts:146]: Thanks!

**Comment by dario-piotrowicz** [comment]
> LGTM Thanks. Just resolve this https://github.com/opennextjs/opennextjs-aws/actions/runs/12351796922/job/34467482276?pr=678

done 🙂👍

**Pull request merged**